### DISCUSSION TITLE
Интервал между элементами

### DISCUSSION THREAD

**Discussion by Xclio** [farag2/Sophia-Script-for-Windows#271] [Q&A] [answered]
приветствую )
в настройках Параметры папок - Вид, в самом низу появился параметр - Уменьшить интервал между элементами. очень полезная опция... ) если можно, внести бы его в скрипт...

**Comment by farag2** [comment, marked as answer]
Hi!
Не оно разве?
https://github.com/farag2/Sophia-Script-for-Windows/blob/23318eada047058a56adad5f831b471c590cb768/Sophia%20Script/Sophia%20Script%20for%20Windows%2011/Sophia.ps1#L313

**Reply by Xclio** [reply]
ааа, я понял, почему упустил это, в Wrapper нет всплывающей подсказки на этот пункт, а при первой установке 11 винды не соотнес, потому как не знал о новой опции ) теперь нашел, спасибо
